organisaatio-ui/src/components/Sivut/Tyypit/Tyypit.tsx

```diff
@@ -29,7 +30,7 @@ const urlPrefix = process.env.NODE_ENV === 'development' ? '/api' : 'organisaati
 
 const Tyypit = (props: Props) => {
     const { i18n, language } = useContext(LanguageContext);
-    const [tyypit, setTyypit] = useState<string[] | YhteystietoTyyppi[] | undefined>(undefined);
+    const [tyypit, setTyypit] = useState<YhteystietoTyyppi[] | undefined>(undefined);
```

Comment:
why accepting undefined? consider using empty array as default value

Reply:
i will try to change this accordingly